### PR TITLE
extensibility: document cross-functional review process

### PR DESCRIPTION
**Pull request by tjkandala**
Context: we decided in a [retro meeting](https://docs.google.com/document/d/1uLR_2ICQB3jSCtQ3ebpQqvUYg11IIm0ZRJ5LoZ_wp2M/edit#bookmark=id.z2i6trrf5zgf) that we should formalize how extensibility engineers ask cross-functional teammates for review.

### CODE REVIEW

**Comment by tjkandala** [comment]
@Joelkw @AlicjaSuska Does the process outlined in this PR work for you? (review by end of next week, low priority)

File: handbook/engineering/developer-insights/extensibility/index.md [hunk 1]
```diff
@@ -129,6 +136,10 @@ The team follow's the [default code review guidelines](https://docs.sourcegraph.
 1. If the author would like any of the requested reviewers to merge the PR after approval they add the label `merge-on-any-approve`
 1. If the author would like their PR to be merged once all of the requested reviewers have approved it they add the label `merge-on-all-approve`
 
+#### Cross-functional reviews
+
+Engineers will often need reviews from teammates from other functions. For example, a pull request can benefit from visual review from a designer. In such cases, the author should be sure to tag the requested cross-functional reviewer(s) in their PR description, along with what exactly they need reviewed and when they need it to be reviewed by (e.g. "@Joelkw can you review the banner copy by the end of the week?").
```
Comment (AlicjaSuska) [inline handbook/engineering/developer-insights/extensibility/index.md:141]:
```suggestion
Engineers will often need reviews from teammates from other functions. For example, a pull request can benefit from visual review from a designer. In such cases, the author should be sure to request the PR review as well as tag the requested cross-functional reviewer(s) in their PR description, along with what exactly they need reviewed and when they need it to be reviewed by (e.g. "@Joelkw can you review the banner copy by the end of the week?").
```

Comment (AlicjaSuska) [reply handbook/engineering/developer-insights/extensibility/index.md:141]: @tjkandala I'd say we need both:
- request the review
- tag someone to describe what to review and by when
Thank you for making thos changes!

**Review by AlicjaSuska — CHANGES_REQUESTED** [review]

File: handbook/engineering/developer-insights/extensibility/index.md (hunk 1 above)
Comment (tjkandala) [reply handbook/engineering/developer-insights/extensibility/index.md:141]: Thanks!

File: handbook/engineering/developer-insights/extensibility/index.md [hunk 2]
```diff
@@ -129,6 +136,10 @@ The team follow's the [default code review guidelines](https://docs.sourcegraph.
 1. If the author would like any of the requested reviewers to merge the PR after approval they add the label `merge-on-any-approve`
 1. If the author would like their PR to be merged once all of the requested reviewers have approved it they add the label `merge-on-all-approve`
 
+#### Cross-functional reviews
+
+Engineers will often need reviews from teammates from other functions. For example, a pull request can benefit from visual review from a designer. In such cases, the author should be sure to request the PR review as well as tag the requested cross-functional reviewer(s) in their PR description, along with what exactly they need reviewed and when they need it to be reviewed by (e.g. "@Joelkw can you review the banner copy by the end of the week?").
```
Comment (felixfbecker) [inline handbook/engineering/developer-insights/extensibility/index.md:141]: Might be worth linking to https://about.sourcegraph.com/handbook/communication/seeking-and-giving-feedback#seeking-feedback, which applies here

Comment (tjkandala) [reply handbook/engineering/developer-insights/extensibility/index.md]: Thanks!

**Comment by tjkandala** [comment]
@muratsu @Joelkw I'm merging this as it's been open for over a week, feel free to do a post-merge review

**Pull request merged**